test/runtests.jl

```diff
@@ -89,7 +89,7 @@ end
 @testset "dates_1" begin
     jd = @jimport(java.util.Date)(())
     jcal = @jimport(java.util.GregorianCalendar)(())
-    jsd =  @jimport(java.sql.Date)((jlong,),round(jlong, time()))
+    jsd =  @jimport(java.sql.Date)((jlong,),round(jlong, digits=time()))
```

Comment:
bad bot

Reply:
FemtoCleaner didn't recognize `jlong` as a type and proposed incorrect signature change.